source/initHQ/fortifyFOB.sqf

```diff
@@ -1,12 +1,17 @@
-_centerPos = _this select 3;
+_centerPos = (_this select 3) select 0;
 _actionID = _this select 2;
 _object = _this select 0;
+_fob = (_this select 3) select 1;
+aliveAllUnits = {alive _x} count allunits;
 
 if (commandpointsblu1 < 4) exitWith {hint "You don't have enough Command Points"};
 commandpointsblu1 = commandpointsblu1 - 4;
 publicvariable "commandpointsblu1";
-_object removeAction _actionID;
 
+
+if ((Warcom_Limiter_Param == 1) && (aliveAllUnits>99)) exitWith {hint "*DUWS AI Limiter is ENABLED!*\n\nTry Fortifying again when there are less than 100 AI units on the map"};
```

Comment:
No no. What I meant is having the 99 in a variable that is configurable globally. That way people with lower end machines could have a limit of 50 AI. People with average could keep 100 and high end beefy machines could use 150 or 200. That's all completely hypothetical. 

Basically I was thinking...
```
unitCap = 100;
if ((Warcom_LImiter_Param == 1) && (aliveAllUnits >= unitCap)) exitWith {...
```